.github/workflows/clang-tidy.yml

```diff
@@ -35,4 +35,4 @@ jobs:
     - name: run
       shell: bash
       working-directory: ${{ runner.workspace }}/_build
-      run: run-clang-tidy -checks=*,-clang-analyzer-deadcode*,-clang-analyzer-optin*
+      run: run-clang-tidy -config-file=${{ runner.workspace }}/.clang-tidy
```

Comment:
It will be picked up automatically

Reply:
it's not finding it, which suggests the same as when i passed it explicitly: that it's not running with the root as a parent directory, somehow.

Reply:
it finds it locally, fwiw